src/sys/socket/mod.rs

```diff
@@ -1552,11 +1552,11 @@ pub fn sendmmsg<'a, XS, AS, C, I, S>(
     flags: MsgFlags
 ) -> crate::Result<MultiResults<'a, S>>
     where
-        XS: IntoIterator<Item = &'a I>,
```

Comment:
Could you explain the motivation for this part?

Reply:
`AsRef<U>` is implemented for `&T` if `T: AsRef<U>`, which means that this is less restrictive and all previous function calls should be compatible with this new signature (while this still being a breaking change).

The additional reference is simply redundant and could be a hurdle when trying to create the iterator.